BIJ12 testset geeft fout, 2.2-vrn-0.3.1 op acc

Ik heb gebruiker RijkswaterstaatTest aangemaakt en de volgende dataset toegevoegd:
http://ftp.geodan.nl/vrn/data/test/GML_test_tbv_CLV_08012015/GebiedBeheerNM.gml;http://ftp.geodan.nl/vrn/data/test/GML_test_tbv_CLV_08012015/GebiedBeheerNM.xsd;GebiedBeheerNM

Bij valideren en importeren geeft hij de volgende fout: 
Fout bij parsen Features. Oorzaak: NoResultException: No entity found for query

13:20:12,853 DEBUG DBLogger:29 - Fout bij parsen Features. Oorzaak: NoResultException: No entity found for query  (XML_FEATURES_EXCEPTION)
13:20:12,853 ERROR AbstractProcess:220 - Error while parsing/processing features
org.springframework.dao.EmptyResultDataAccessException: No entity found for query; nested exception is javax.persistence.NoResultException: No entity found for query
        at org.springframework.orm.jpa.EntityManagerFactoryUtils.convertJpaAccessExceptionIfPossible(EntityManagerFactoryUtils.java:304)
        at org.springframework.orm.jpa.aspectj.JpaExceptionTranslatorAspect.ajc$afterThrowing$org_springframework_orm_jpa_aspectj_JpaExceptionTranslatorAspect$1$18a1ac9(JpaExceptionTranslatorAspect.aj:33)
        at nl.ipo.cds.dao.impl.ManagerDaoImpl.getBronhouderGeometry(ManagerDaoImpl.java:922)
        at nl.ipo.cds.etl.theme.vrn.validation.AbstractVrnValidator.beforeJob(AbstractVrnValidator.java:112)
        at nl.ipo.cds.etl.theme.vrn.validation.AbstractVrnValidator.beforeJob(AbstractVrnValidator.java:52)
        at nl.ipo.cds.etl.AbstractValidator.getFilterForJob(AbstractValidator.java:313)
        at nl.ipo.cds.etl.AbstractValidator.getFilterForJob(AbstractValidator.java:279)
        at nl.ipo.cds.etl.AbstractValidator.getFilterForJob(AbstractValidator.java:252)
        at nl.ipo.cds.etl.process.ValidateFeatureProcessor.createFilters(ValidateFeatureProcessor.java:91)
        at nl.ipo.cds.etl.process.ValidateFeatureProcessor.processFeatures(ValidateFeatureProcessor.java:143)
        at nl.ipo.cds.etl.AbstractProcess.processFeatures_aroundBody2(AbstractProcess.java:228)
        at nl.ipo.cds.etl.AbstractProcess$AjcClosure3.run(AbstractProcess.java:1)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect.ajc$around$org_springframework_transaction_aspectj_AbstractTransactionAspect$1$2a73e96cproceed(AbstractTransactionAspect.aj:59)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect$AbstractTransactionAspect$1.proceedWithInvocation(AbstractTransactionAspect.aj:65)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:260)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect.ajc$around$org_springframework_transaction_aspectj_AbstractTransactionAspect$1$2a73e96c(AbstractTransactionAspect.aj:63)
        at nl.ipo.cds.etl.AbstractProcess.processFeatures(AbstractProcess.java:227)
        at nl.ipo.cds.etl.AbstractProcess.processFeatures(AbstractProcess.java:201)
        at nl.ipo.cds.etl.AbstractProcess.process(AbstractProcess.java:151)
        at nl.ipo.cds.etl.AbstractProcess.process(AbstractProcess.java:123)
        at nl.ipo.cds.etl.AbstractProcess.process_aroundBody0(AbstractProcess.java:119)
        at nl.ipo.cds.etl.AbstractProcess$AjcClosure1.run(AbstractProcess.java:1)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect.ajc$around$org_springframework_transaction_aspectj_AbstractTransactionAspect$1$2a73e96cproceed(AbstractTransactionAspect.aj:59)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect$AbstractTransactionAspect$1.proceedWithInvocation(AbstractTransactionAspect.aj:65)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:260)
        at org.springframework.transaction.aspectj.AbstractTransactionAspect.ajc$around$org_springframework_transaction_aspectj_AbstractTransactionAspect$1$2a73e96c(AbstractTransactionAspect.aj:63)
        at nl.ipo.cds.etl.AbstractProcess.process(AbstractProcess.java:118)
        at nl.ipo.cds.etl.AbstractProcess.process(AbstractProcess.java:1)
        at nl.idgis.commons.jobexecutor.JobProcessor$1.call(JobProcessor.java:73)
        at nl.idgis.commons.jobexecutor.JobProcessor$1.call(JobProcessor.java:66)
        at nl.idgis.commons.jobexecutor.JobExecutor.processJob(JobExecutor.java:120)
        at nl.idgis.commons.jobexecutor.JobExecutor.run(JobExecutor.java:89)
        at nl.ipo.cds.executor.CdsJobExecuter.run(CdsJobExecuter.java:92)
        at nl.ipo.cds.executor.CdsJobExecuter.main(CdsJobExecuter.java:72)
Caused by: javax.persistence.NoResultException: No entity found for query
        at org.hibernate.ejb.QueryImpl.getSingleResult(QueryImpl.java:280)
        ... 32 more

--- a/managerDB/src/main/java/nl/ipo/cds/dao/impl/ManagerDaoImpl.java
+++ b/managerDB/src/main/java/nl/ipo/cds/dao/impl/ManagerDaoImpl.java
@@ -922,8 +922,10 @@ public class ManagerDaoImpl implements ManagerDao {
 			value = (byte[])query.getSingleResult ();
 		} catch (NoResultException e) {
 			return null;
+		} catch (EmptyResultDataAccessException e) {
+			return null;
 		}
-		
+
 		try {
 			return WKBReader.read (value, null);
 		} catch (ParseException e) {

